content/en/docs/tasks/administer-cluster/kubelet-credential-provider.md

```diff
@@ -54,7 +54,7 @@ In order to use this feature, the kubelet expects two flags to be set:
 
 The configuration file passed into `--image-credential-provider-config` is read by the kubelet to determine which exec plugins
 should be invoked for which container images. Here's an example configuration file you may end up using if you are using the
-[ECR](https://aws.amazon.com/ecr/)-based plugin:
+[ECR-based plugin](https://github.com/kubernetes/cloud-provider-aws/tree/master/cmd/ecr-credential-provider):
```

Comment:
Even better, if feasible, is to link to the AWS documentation for configuring this.

Do we know if there's anything to link to?

Reply:
The mkdocs don't have anything useful: https://cloud-provider-aws.sigs.k8s.io/#aws-credential-provider

My call was: it's better to link to the cred provider implementation versus some other doc

Reply:
https://cloud-provider-aws.sigs.k8s.io/#aws-credential-provider is actually Kubernetes documentation, not AWS documentation.

Sounds like there's no AWS docs on this detail.